MIGRATING.md

```diff
@@ -0,0 +1,77 @@
+# Okta Java Management SDK Migration Guide
+ 
+This SDK uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
+
+## Migrating from 1.x.x to 2.0.0
+
+Version 2.0.0 of this SDK introduces a number of breaking changes from previous versions. 
+In addition to many new classes/interfaces, some existing class/interfaces are no longer backward compatible due to method renaming and signature changes, see [Breaking Changes](#breaking-changes).
+
+### Breaking Changes
+
+Following is a list of changes that break backward compatibility in version 2.0.0.
+
+- `com.okta.sdk.client.Client`
+   - Renamed `createRule` to `createGroupRule`
+   - Renamed `getRule` to `getGroupRule`
+   - Renamed `listRules` to `listGroupRules`
+   
+- `com.okta.sdk.resource.log.LogEventList` 
+   - `getLogs(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)`
+   Signature changed to `getLogs(java.util.Date, java.util.Date, java.lang.String, java.lang.String, java.lang.String)`
+   
+- `com.okta.sdk.resource.group.GroupList` 
+   - `listGroups(java.lang.String,java.lang.String, java.lang.String)`
+   Signature changed to `listGroups(java.lang.String, java.lang.String)`
+
+- `com.okta.sdk.resource.application.JsonWebKey`
+   - `getExpiresAt` Return type changed from `Date` to `String`
```

Comment:
What? That seems like a bug (and a bad one)

Reply:
Fixed this.